Scheduler/Pages/WeatherPage.xaml

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Scheduler.Pages.WeatherPage"
+             xmlns:views="clr-namespace:Scheduler.Views"
+             BackgroundImage="weatherBackground.jpg">
+    <ContentPage.Content>
+        <StackLayout>
```

Comment:
for your information, if you have only one View - you can put it without 'parent' StackLayout